tests/Application/Commands/HandleOutOfSyncFundsTest.php

```diff
@@ -19,6 +19,18 @@
 
 class HandleOutOfSyncFundsTest extends TestCase
 {
+    private CampaignFunding $fundingUnderMatched;
+    private CampaignFunding $fundingInSync;
+    private CampaignFunding $fundingOverMatched;
+    private CampaignFunding $fundingUnderMatchedWithNothingAllocated;
+
+    public function setUp(): void
+    {
+        $this->fundingUnderMatched = $this->getFundingUnderMatched();
+        $this->fundingInSync = $this->getFundingInSync();
+        $this->fundingOverMatched = $this->getFundingOverMatched();
+        $this->fundingUnderMatchedWithNothingAllocated = $this->getFundingUnderMatchedWithNothingAllocated();
```

Comment:
as these objects now contain timestamps we can't construct them multiple times and expect them to be equal. If this test was left unchanged it would be failing. So constructing once in setUp instead of having multiple calls to these get functions.